docs/about/themes-translation.md

```diff
@@ -75,6 +75,43 @@ mkdocs/themes/readthedocs/locales
 │       └── messages.po
 ```
 
+You can now move on to the next step and [add a translation](#translating-the-mkdocs-themes)
+for every text placeholder in the localized catalogs.
+
+## Updating a theme translation
+
+If a [theme's `messages.pot` template file has been updated](/user-guide/custom-themes/#localizing-themes)
+since the `messages.po` was last updated for your locale, follow the steps below
+to update the theme's `messages.po` file:
+
+1. [Translate](#translating-the-mkdocs-themes) the newly added translatable text
+placeholders on every `messages.po` catalog file language you can
+2. [Serve and test locally](#testing-themes-translations) the translated themes
+for your language
+3. [Contribute your updated translations](#contributing-your-translations)
+through a Pull Request
```

Comment:
Don't we need to add a new step one here: "Updating the translation catalogs"? And then the existing step 1 would become step 2, etc.

To be clear, it appears that the explanation of that step is in the proper location in the documentation. However it was simply omitted from this list.

Other than omitting the step from the list. I think this looks good.

Reply:
> Don't we need to add a new step one here: "Updating the translation catalogs"? And then the existing step 1 would become step 2, etc.

Absolutely, sorry for missing it. Fixed!